worker/sri.txt

```diff
@@ -1 +1 @@
-{"__version": 200, "updater.py": "PHFUVXcoxBFiW2hTqFN5q5WdAw2pK8uzFC7hyMUC3cLY5bGZPhL8TBGThtqDmcXd", "worker.py": "bYP5sBennlrsURZiN0CwotQLYapRMDL4CIG/2sfyzQL/1fuzl1ZsQDdawOM/b92j", "games.py": "wzzUVbeYyxLGRQTxx0RyCh67lHVadBe6HIQi84GM2W27cF3+aJIXLZkZmysfxTv5"}
+{"__version": 200, "updater.py": "PHFUVXcoxBFiW2hTqFN5q5WdAw2pK8uzFC7hyMUC3cLY5bGZPhL8TBGThtqDmcXd", "worker.py": "bYP5sBennlrsURZiN0CwotQLYapRMDL4CIG/2sfyzQL/1fuzl1ZsQDdawOM/b92j", "games.py": "wzzUVbeYyxLGRQTxx0RyCh67lHVadBe6HIQi84GM2W27cF3+aJIXLZkZmysfxTv5"}
```

Comment:
I fixed the conflict from GitHub GUI to be able to use the PR in the test workers.